.fixtures.yml

```diff
@@ -2,9 +2,9 @@ fixtures:
   repositories:
       stdlib:
         repo: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
-        branch: "4.13.1"
```

Comment:
please switch to git branches and not a specific tag. in unit tests we test against unreleased changes, in acceptance tests we use forge releases.